test/test_dtype.py

```diff
@@ -53,7 +53,12 @@ def _test_op(fxn, target_dtype:DType, target):
 def _test_cast(a:Tensor, target_dtype:DType):
   _test_op(lambda: a.cast(target_dtype), target_dtype, list(a.numpy().astype(target_dtype.np)))
 def _test_bitcast(a:Tensor, target_dtype:DType, target=None):
-  _test_op(lambda: a.bitcast(target_dtype), target_dtype, target or a.numpy().view(target_dtype.np).tolist())
+  if Device.DEFAULT=='PYTHON':
```

Comment:
Why?

Reply:
This gets executed before the bit cast: a.numpy().view(target_dtype.np).tolist()

Reply:
This is not resolved? Why should PYTHON backend be different?